-meeting-template.md

```diff
@@ -8,31 +8,43 @@ filename: -meeting-template.md
 [:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
 
 Time: ...  
-Location: https://meet.jit.si/offline  
+Location: https://meet.jit.si/hyphacoop  
 Attending: ...
 
 [TOC]
 
 ## Agenda
 
+- Previous Action Items
+- Runthrough Announcements and Updates (add below)
 - Quick Working Group Updates (10 mins)
     - [Business Planning][biz-wg]
     - [Finance][fin-wg]
     - [Governance][gov-wg]
     - [Infra & Ops][ino-wg]
+- Discussion
+    ...
+
+## Announcements
+
 - ...
 
 ## Notes
 
 - ...
 
-## Actionable Items
+## Follow Up
+
+### Actionable Items
 
 - [ ] ...
 
+### Upcoming Discussions
+
+- ...
 
 <!-- Links -->
-[template]: https://hackmd.io/0mgHiik2QwOLbOT-2_Uh7w?edit
+[template]: https://link.hypha.club/template
```

Comment:
```suggestion
[template]: https://link.hypha.coop/template
```

Reply:
None of our internal tools are using the systems/codes/club domain, so I put the shortlink experiment on a long-term domain